src/triagebot/no-mentions.md

```diff
@@ -0,0 +1,17 @@
+# No Mentions (in commits)
+
+GitHub permits having mentions (eg. `@user`) in commits, and while it's sometimes useful it's almost ended-up spamming the mentioned users, in particular as the commits are being rebased, cherry-picked or pushed.
```

Comment:
`it's almost ended-up` - this sounds like it's missing something?

Reply:
yeah, should be: `it almost always ends-up`